Mathlib/CategoryTheory/Preadditive/Mat.lean

```diff
@@ -341,7 +341,7 @@ def isoBiproductEmbedding (M : Mat_ C) : M ≅ ⨁ fun i => (embedding C).obj (M
     rw [Finset.sum_apply, Finset.sum_apply, Finset.sum_eq_single i]; rotate_left
     · intro b _ hb
       dsimp
-      simp only [Finset.sum_singleton]
+      erw [Finset.sum_singleton]
```

Comment:
Can we use `rw [Finset.univ_ofSubsingleton, Finset.sum_singleton]` here? I'd like to reduce usage `erw` if possible.

Reply:
Yes indeed! I tried turning it on as an instance here so assumed something else was going on. But it is good. Thanks.